cob_gazebo_worlds/urdf/empty/empty.urdf.xacro

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro"
+       name="empty">
+
+  <link name="world"/>
```

Comment:
So empty does relly only spawn the world link and not even a floor? Or did I miss something?